rkt/cli_apps.go

```diff
@@ -246,3 +246,91 @@ func (al *appsVolume) String() string {
 	}
 	return strings.Join(vs, " ")
 }
+
+// A flag value type supporting a csv list of options
+type OptionList struct {
```

Comment:
Hmm, how about moving those to separate package? I did some refactoring of `rkt/images.go` in #1763, so if I'm going to rebase this work on top of yours then I have two options:
- Explode the option list into a bunch of bools image fetching would be interested in
- Just take a SecList variable
  - This would need putting SecList in a separate package to avoid circular package imports.